gravitee-apim-portal-webui/src/app/components/gv-alert/gv-alert.component.html

```diff
@@ -104,6 +104,19 @@ <h3 class="title">{{ 'application.alerts.add.title' | translate }}</h3>
           ngDefaultControl
         ></gv-select>
         <strong *ngIf="isReadingMode"> {{ timeUnitTranslated() }} </strong>
+        <div>
+          <span class="gv-alert__description-text" *ngIf="isReadingMode">{{ replaceToLineBreak(alert['description']) }}</span>
+          <gv-text
+            *ngIf="isCreationMode || isEditionMode"
+            ngDefaultControl
+            class="gv-alert__description-input"
+            formControlName="description"
+            name="description"
+            rows="3"
+            label="Description"
```

Comment:
Maybe label should also be translated (even if it's identifier for fr/en, it's not for cs ?)